apps/frontend-control/.env.production

```
@@ -5,11 +5,22 @@ NEXT_PUBLIC_AVATAR_BASE_PATH="https://sos-ch-dk-2.exo.io/klicker-prod/avatars"
-NEXT_PUBLIC_PWA_URL="https://pwa.klicker.uzh.ch"
-NEXT_PUBLIC_MANAGE_URL="https://manage.klicker.uzh.ch"
-NEXT_PUBLIC_CONTROL_URL="https://control.klicker.uzh.ch"
-NEXT_PUBLIC_AUTH_URL="https://auth.klicker.uzh.ch"
-NEXT_PUBLIC_LTI_URL="https://lti.klicker.uzh.ch"
-
 NEXT_PUBLIC_MATOMO_URL="https://webstats.uzh.ch/"
 NEXT_PUBLIC_MATOMO_SITE_ID=480
+
+APP_ORIGIN_API=https://backend-sls.klicker.uzh.ch
+APP_ORIGIN_PWA=https://pwa.klicker.uzh.ch
+APP_ORIGIN_MANAGE=https://manage.klicker.uzh.ch
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The APP_ORIGIN_MANAGE key should go before the APP_ORIGIN_PWA key

(UnorderedKey)